<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPvdbCreateAssessmentPartialDialog</class>
 <widget class="QDialog" name="QtPvdbCreateAssessmentPartialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formuleer de focusvragen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Brainweaver.qrc">
    <normaloff>:/images/PicArrow14x14.png</normaloff>:/images/PicArrow14x14.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Geef vier focussen, drie voor het associëren en een voor de complete concept map</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Eerste focus om op te associëren:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_1">
     <property name="text">
      <string>Als docent kan ik ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tweede focus om op te associëren:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_2">
     <property name="text">
      <string>Als docent ben ik als mens ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Derde focus om op te associëren</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_3">
     <property name="text">
      <string>Als docent weet ik van leerlingen ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Focus voor de complete concept map:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_complete">
     <property name="text">
      <string>Wat voor een docent wiskunde ben ik?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_save">
     <property name="text">
      <string>&amp;Sla assessment bestand op</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Brainweaver.qrc"/>
 </resources>
 <connections/>
</ui>
